src/components/section/ohri-form-section.component.tsx

```diff
@@ -75,16 +75,16 @@ const OHRIFormSection = ({ fields, onFieldChange }) => {
                       className={`${
                         fieldDescriptor.questionOptions.rendering == 'radio' ||
                         fieldDescriptor.questionOptions.rendering == 'date' ||
-                        fieldDescriptor.questionOptions.rendering == 'datetime'
+                        fieldDescriptor.questionOptions.rendering == 'datetime' ||
+                        fieldDescriptor.questionOptions.rendering == 'number'
```

Comment:
@arodidev can you consider using [classnames](https://github.com/JedWatson/classnames#readme) to conditionally apply CSS classes? The other thing is we should be more explicit on what control (determined by the rendering) the  `styles.flexBasisOn` class is targeting; this could help minimise bugs.